doc/en/announce/release-7.0.0rc1.rst

```diff
@@ -0,0 +1,88 @@
+pytest-7.0.0rc1
+=======================================
+
+pytest 7.0.0rc1 has just been released to PyPI.
+
+This is a bug-fix release, being a drop-in replacement. To upgrade::
+
+  pip install --upgrade pytest
+
+The full changelog is available at https://docs.pytest.org/en/stable/changelog.html.
+
+Thanks to all of the contributors to this release:
+
+* Abdelrahman Elbehery
+* Adam Johnson
+* Albert Villanova del Moral
+* Alexandros Tzannes
+* Andreas Motl
+* Anthony Sottile
+* Anton
+* Benjamin Wohlwend
+* Bruno Oliveira
+* Chris NeJame
+* Chris Rose
+* Christian Steinmeyer
+* Christophe Bedard
+* Daniele Procida
+* Denis Laxalde
+* Dominic Davis-Foster
+* Feanil Patel
+* Florian Bruhin
+* Gergely Imreh
+* Hong Xu
+* Jakob van Santen
+* James Bourbeau
+* James Gerity
+* Jason Haugen
+* Jeff Widman
+* Johannes Maron
+* Jürgen Gmach
+* Kale Kundert
+* MapleCCC
+* Matthew Hughes
+* Maximilian Cosmo Sitter
+* Mehera Emrich
+* Miro Hrončok
+* Miroslav Šedivý
+* Nico Schlömer
+* Olga Matoula
+* Parth Patel
+* Pax
+* Petr Viktorin
+* Pierre Mourlanne
+* Rahul Kumaresan
+* Ran Benita
+* Ronny Pfannschmidt
+* Ross Vandegrift
+* Saiprasad Kale
+* Simon K
+* Srip
+* Stefaan Lippens
+* Sylvain Bellemare
+* Tadeu Manoel
+* Tarcísio Fischer
+* Thomas Grainger
+* Vincent Poulailleau
+* Zac Hatfield-Dodds
+* Zack Kneupper
+* alshapton
+* antonblr
+* bengartner
+* bluetech
+* bot2x
+* dependabot[bot]
```

Comment:
Not related to the release itself, but we should probably remove `[bot]` names here.